cmake/projects/odb-compiler/hunter.cmake

```diff
@@ -29,9 +29,9 @@ elseif(CMAKE_HOST_UNIX)
       VERSION
       "2.4.0"
       URL
-      "http://www.codesynthesis.com/download/odb/2.4/odb-2.4.0-i686-linux-gnu.tar.bz2"
```

Comment:
> is dependent on the target as far as I understood it

Okay, then `CMAKE_SIZEOF_VOID_P` should not be used here. I don't know what is the best way to solve it. I've searched a little bit and found [CMAKE_HOST_SYSTEM_PROCESSOR](https://cmake.org/cmake/help/latest/variable/CMAKE_HOST_SYSTEM_PROCESSOR.html) variable, it return `x86_64` on my Linux.

One more note is that using `url_sha1_unpack_install` scheme is not quite correct since it mix host and target executables. I haven't designed this part in Hunter for now, planning to make some improvements in future.